go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21.0
 require (
 	github.com/google/go-cmp v0.6.0
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/hashicorp/hcl/v2 v2.19.1
+	github.com/hashicorp/hcl/v2 v2.20.0
```

Comment:
this comment serves as a blocker as per https://github.com/hashicorp/hcl-lang/pull/379#issuecomment-1972851690